packages/system/redis-operator/Makefile

```
@@ -9,3 +11,16 @@ update:
 	helm repo update redis-operator
 	helm pull redis-operator/redis-operator --untar --untardir charts
 	sed -i '/{{/d' charts/redis-operator/crds/databases.spotahome.com_redisfailovers.yaml
+
+image:
```

checkmake (0.2.2)

[warning] Target body for "image" exceeds allowed length of 5 (11).

(maxbodylength)